client/src/styles.css

```diff
@@ -43,37 +45,33 @@ body {
 }
 
 .right.visible {
-  position:fixed;
-  overflow: hidden;
-  left:77%;
-  top: 27%;
+  position: relative;
+  top: 55px;
   opacity: 10;
+  height: 60px;
 }
 
 .left.visible {
-  position:fixed;
-  overflow: hidden;
-  top: 27%;
-  left: 13%;
+  position: relative;
+  top: 55px;
   opacity: 10;
+  height: 60px;
 }
 
 .right.invisible {
-  visibility: hidden;
+  visibility: visible;
+  position: relative;
   opacity: 10;
-  position:fixed;
-  overflow: hidden;
-  left:77%;
-  top: 27%;
+  top: 55px;
+  height: 60px;
 }
 
 .left.invisible {
-  visibility: hidden;
+  visibility: visible;
```

Comment:
What happens to the arrow when the scrollbar reaches the end? Is visibility changed? They're supposed to be invisible.

Reply:
Good catch. That was for my testing. I'll push a fix.